matrix-sdk/src/main/java/org/matrix/androidsdk/data/Room.java

```diff
@@ -183,6 +183,16 @@ public IMXStore getStore() {
         return mStore;
     }
 
+    /**
+     * Determine whether we should encrypt messages for invited users in this room.
+     *
+     * @return true if we should encrypt messages for invited users.
+     */
+    public boolean shouldEncryptForInvitedMembers() {
```

Comment:
From the Room class point of view, this method is not related to encryption.
I would rename it to `isHistoryVisibleForNotJoinedMembers` or something similar
Please also update the Javadoc accordingly

Reply:
ok